.github/workflows/optimize-e2e-test-cloud.out

```diff
@@ -0,0 +1,106 @@
+# NOTE:
+# This pipeline is currently not enabled. For more info, see:
+# https://github.com/camunda/camunda/issues/24886
+
+name: Optimize e2e cloud test
+
+on:
+  pull_request:
+    paths:
+      - ".github/actions/**"
+      - ".github/workflows/optimize-*"
+      - "bom/*"
+      - "parent/*"
+      - "pom.xml"
+      - "optimize/**"
+      - "optimize.Dockerfile"
+  push:
+    branches:
+      - "main"
+      - "stable/**"
+      - "release/**"
+    paths:
+      - ".github/actions/**"
+      - ".github/workflows/optimize-*"
+      - "bom/*"
+      - "parent/*"
+      - "pom.xml"
+      - "optimize/**"
+      - "optimize.Dockerfile"
+  workflow_dispatch:
+
+jobs:
+  e2e-cloud-test:
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
+      - name: Import secrets
+        id: secrets
+        uses: hashicorp/vault-action@8b7eaceb79bd7be97b311993e7c1cc5d051213c7
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/optimize/ci/camunda-optimize AUTH0_CLIENTSECRET;
+            secret/data/products/optimize/ci/camunda-optimize AUTH0_USEREMAIL;
+            secret/data/products/optimize/ci/camunda-optimize AUTH0_USERPASSWORD;
+      - name: Setup Maven
+        uses: ./.github/actions/setup-build
+        with:
+          java-distribution: zulu
+          maven-cache-key-modifier: optimize-tests
+          maven-version: 3.8.6
+          time-zone: Europe/Berlin
+          vault-address: ${{ secrets.VAULT_ADDR }}
+          vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+          vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      - name: Setup Chrome
+        shell: bash
+        run: |
+          # remove existing chrome
+          sudo apt-get remove -y google-chrome-stable
+          # install chrome 126
+          wget https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_126.0.6478.182-1_amd64.deb
+          sudo apt-get update && sudo apt-get install -y ./google-chrome-stable_126.0.6478.182-1_amd64.deb
```

Comment:
I know this is a copy/paste from the other workflow, and this is not a blocker, but is this going to be easily maintainable? Chrome 126 will _eventually_ go out of support. Is there a better way to get a good version of Chrome rather than hard-coding it?

Reply:
I think we can create an issue for this and resolve it in another PR, or even fix it when we decide to upgrade chrome?